wordpress-mobile/WordPress-iOS
# Fix a crash caused by cellForRowAtIndexPath method in BlogDetailsViewController returning a nil cell

Fixes #20835

## Description

This PR addresses a crash that occurs on the Blog Details screen, which stems from a particular edge case. The issue arises when the following method:

```objective-c
- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
```

returns a `nil` cell. In this case, the expected return value should be a valid `UITableViewCell`, but instead, `nil` is returned, leading to a crash.

## Test Instructions

I managed to reproduce the crash iPad only. However, for comprehensive testing, I recommend that the following steps be carried out on both iPhone and iPad devices.

1. Clean install both WordPress and Jetpack.
2. On WordPress, log into your account.
3. Perform Jetpack Migration.
4. Make sure you're on **My Site** tab and the **Please Delete WordPress** cell is visible.
5. Close the Jetpack app ( by moving it to background ).
6. Delete WordPress.
7. Open Jetpack app again.
8. **Expect** the app not to crash and **Please Delete WordPress** cell is no longer visible.
 
## Regression Notes
1. Potential unintended areas of impact
N/A

9. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

10. What automated tests I added (or what prevented me from doing so)
N/A

## PR submission checklist:
